orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -499,9 +499,20 @@ at::Tensor _copy_from_and_resize(
   const auto ort_self = create_ort_value(invoker, self);
   auto ort_dst = create_ort_value(invoker, dst);
 
+  // If the size of the dst is not equal to src, re-allocate the dst tensor to
+  // match the src tensor size.
+  if (self.numel() != dst.numel()) {
+    auto* dst_ort_tensor = ort_dst.GetMutable<onnxruntime::Tensor>();
+    onnxruntime::Tensor::InitOrtValue(dst_ort_tensor->DataType(), onnxruntime::TensorShape(self.sizes().vec()),
+      invoker.GetCurrentExecutionProvider().GetAllocator(0, OrtMemTypeDefault), ort_dst);
+
+    auto* dst_impl = dynamic_cast<ORTTensorImpl*>(dst.unsafeGetTensorImpl());
+    dst_impl->set_tensor(ort_dst);
+  }
+
   copy(invoker, ort_self, ort_dst);
 
-  return self;
+  return dst;
```

Comment:
do you know what tensor this is expecting returned? As in is this documented somewhere?

Reply:
I was not able to find much / any documentation. I did check with what other implementations return fpr reference, and it was dst